R/utils_render_register_general.r

```diff
@@ -153,6 +150,13 @@ generate_table_details <- function(table_key, table, filter, is_reg_table = TRUE
 #'
 #' @return None. The function generates a file in the specified format.
 render_register <- function(register_table, table_details, filter, output_type){
+  # Filtering the columns of the table
+  file_type <- switch(output_type,
```

Comment:
You can also provide a default argument in the function definition:

```R
render_register <- function(register_table, table_details, filter, output_type = "other"){
```

This way you can simply pass on the `output_type`, and every time you need "other", you do not provide the final argument to the function.

Reply:
Good point, completely missed that. Thanks!